.nvmrc

```diff
@@ -1 +1 @@
-6.10.0
+13.3.0
```

Comment:
we should use 10.17.0. 'node-sass' gives ~sass~ with version 13.3.0.

Reply:
Thanks! Great catch, I'll change that and merge.